tests/OverworldDiscordBot.py

```diff
@@ -0,0 +1,22 @@
+# Needs a discord bot account and a server with all the mark ids as emojis
+
+import asyncio
+import io
+import json
+import sys
+from functools import lru_cache
+from threading import Thread
+from time import sleep
+
+import discord
+import pokebase as pb
+from discord.ext import commands
+from discord.utils import get
+from PIL import Image
```

Comment:
unused imports, only ``import json`` and ``import sys`` are needed

Reply:
Fixed.